### ISSUE TITLE
Venezuelan state correction (Carabobo instead of Corabobo)

### ISSUE THREAD

**Issue by EdoRguez** [amcharts/amcharts4#1436]
I'm from Venezuela and I'm using amchart venezuelan map.

```
<script src="https://www.amcharts.com/lib/4/core.js"></script>
<script src="https://www.amcharts.com/lib/4/maps.js"></script>
<script src="https://www.amcharts.com/lib/4/geodata/venezuelaLow.js"></script>
```

The problem is that one of the states is not written properly, this is causing some apps crashing due to this issue.

It should be _"Carabobo"_ instead of _"Corabobo"_.

Regards.

**Comment by martynasma** [comment]
Thanks for the heads up!

This will be fixed with the release of the next version of geodata.

**Comment by martynasma** [comment]
Hey just released a new geodata 4.1.7.

## [4.1.7] - 2019-07-31

### Added
- New maps: Gabon, Kenya.

### Fixed
- Fixed map: `ukCounties` map has Scotland wrong.
- Fixed map: `venezuela` map had a typo in name of "Carabobo".
- Fixed map: `worldRegions*` map had a wrong id for "Africa".
- Typo in `uzbekistan` map file names fixed.
